src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentDictionary.cs

```diff
@@ -979,11 +979,7 @@ public int Count
                     // Acquire all locks
                     AcquireAllLocks(ref acquiredLocks);
 
-                    // Compute the count, we allow overflow
-                    for (int i = 0; i < _tables._countPerLock.Length; i++)
-                    {
-                        count += _tables._countPerLock[i];
-                    }
+                    count = GetCountInternal();
```

Comment:
Yes, it could; I was just trying to keep the code structure as similar to the original implementation as possible. Does anyone else have a preference either way?